crates/bitwarden-uuid-macro/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{
+    parse::{Parse, ParseStream},
+    parse_macro_input, Ident, Visibility,
+};
+
+#[proc_macro]
+pub fn uuid(input: TokenStream) -> TokenStream {
+    // Parse input as: vis ident
+    let input = parse_macro_input!(input as IdTypeInput);
+    let ident = input.ident;
+    let vis = input.vis;
+    let name_str = ident.to_string();
+
+    let tsify_type = format!("Tagged<Uuid, \"{}\">", name_str);
+
+    let expanded = quote! {
+        #[cfg(feature = "wasm")]
+        #[derive(serde::Serialize, serde::Deserialize, tsify_next::Tsify)]
```

Comment:
suggestion: should we use `::serde::Serialize` ?
```suggestion
        #[derive(::serde::Serialize, ::serde::Deserialize, ::tsify_next::Tsify)]
```